var/spack/repos/builtin/packages/amdblis/package.py

```diff
@@ -17,9 +17,44 @@ class Amdblis(BlisBase):
 
     _name = 'amdblis'
     homepage = "https://developer.amd.com/amd-aocl/blas-library/"
-    url = "https://github.com/amd/blis/archive/2.2.tar.gz"
+    url = "https://github.com/amd/blis/archive/3.0.tar.gz"
     git = "https://github.com/amd/blis.git"
 
     maintainers = ['amd-toolchain-support']
 
+    version('3.0', sha256='19ec56cb1c5855a2de22914acfa22fa4e23222992b392c33a2d73a9a8b516667')
     version('2.2', sha256='e1feb60ac919cf6d233c43c424f6a8a11eab2c62c2c6e3f2652c15ee9063c0c9')
+
+    def configure(self, spec, prefix):
```

Comment:
There's a lot of duplication here with `BlisBase`'s `configure`.  Why not add an `configure_args` method here and in `BlisBase` to eliminate the duplication?